index.js

```diff
@@ -188,6 +270,7 @@ module.exports = {
     let userPostTransformPlugins = addonProvidedConfig.postTransformPlugins;
 
     options.plugins = [].concat(
+      shouldIncludeHelpers ** this._getHelpersPlugin(),
```

Comment:
What does this do? I've not seen `**` used like this before...

Reply:
Thaaaat's supposed to be `&&`, this is what I get for submitting code just before bed haha 😅